assets/js/vendor/d3-tip.js

```diff
@@ -0,0 +1,299 @@
+// d3.tip
```

Comment:
We can deal with this separately... we should probably take a more holistic look at how to handle JS dependencies.